### PR TITLE
[CAM-8976] azure brownfield data in alm dashboard

### PR DESCRIPTION
**Pull request by julianatienza**
@gsanchezu Please, can you review these changes? 
@srveit @jim-phillips  would you mind to merge it after the review? 

Thanks so much

### CODE REVIEW

File: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ There are three different submenus under the Instances menu:
 
 * **Unregistered**: shows only unregistered instances, which are the ones accessible into all the defined providers that were not deployed through Cloud Application Manager. They are discovered in the synchronization event of a provider and are classified by Type and Subtype.
   * Type: Any of Compute, Network, Database, Storage or Other
-  * Subtype: this is the instance class, specific to each provider type. For example, for AWS type providers we can see VPC or Application Load Balancers (for Network type instances), and S3 or Elastic Block Storages (for Storage type instances)
+  * Subtype: this is the instance class, specific to each provider type. For example, for AWS type providers we can see VPC or Application Load Balancers (for Network type instances), and S3 or Elastic Block Storages (for Storage type instances) and for the Azure provider we could see Express gateways (in Storage type instances) and Queue or BLOB (in Storage type instances).
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md]:
```suggestion
  * Subtype: this is the instance class, specific to each provider type. For example, for AWS type providers we can see VPC or Application Load Balancers (for Network type instances), and S3 or Elastic Block Storages (for Storage type instances) and for an Azure provider we could see Virtual Networks or Application Gateways (in Network type instances) and Queue or BLOB (in Storage type instances). You can check the [full list of Azure resources being displayed here](./using-microsoft-azure.md#azure-native-resources).
```

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 2]
```diff
@@ -280,6 +281,40 @@ As part of the result of synchronization process you can find a list of availabl
...
 
+### Azure Native Resources
+
+After the synchronization process, you will also find in the Instances page a list of Azure resources that already exist in your account. You can also register virtual machine type instances from here.
+
+![Microsoft Azure - Unregistered instances](../../images/cloud-application-manager/microsoft-azure/unregistered-instances.png)
+
+These resources can be filtered by the following types and subtypes:
+
+* Compute
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]:
```suggestion
* **Compute**
```

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 3]
```diff
@@ -280,6 +281,40 @@ As part of the result of synchronization process you can find a list of availabl
...
+  * ScaleSet
+  * Virtual machine
+* Network
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]:
```suggestion
* **Network**
```

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 4]
```diff
@@ -280,6 +281,40 @@ As part of the result of synchronization process you can find a list of availabl
...
+  * Load balancer
+  * Security groups
+  * Network interface
+  * Virtual networks
+  * IP address
+  * VPN
+  * Application gateway
+  * Local gateway
+  * Express gateway
+* Storage
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]:
```suggestion
* **Storage**
```

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 5]
```diff
@@ -280,6 +281,40 @@ As part of the result of synchronization process you can find a list of availabl
...
+  * File
+  * BLOB
+  * Table
+  * Queue
+* Database
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]:
```suggestion
* **Database**
```

**Comment by gsanchezu** [comment]
Hey @julianatienza, could you please address my comments to approve the changes and allow to publish the doc asap? Thanks!

**Comment by jim-phillips** [comment]
@julianatienza Please resolve conflict.

File: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md [hunk 6]
```diff
@@ -42,8 +42,8 @@ There are three different submenu options under Instances:
 * **Registered**: shows only registered instances, which are the ones deployed through or imported into Cloud Application Manager. In these instances, you can do lifecycle management by executing lifecycle events or using the lifecycle editor to change variables, event scripts or box versions of the instance. You can also select a specific state among the available ones (Online, Unavailable, Processing, Shutdown, Terminated) to show only the instance in the selected state.
 
 * **Unregistered**: shows only unregistered instances, which are the ones accessible into all the defined providers that were not deployed through Cloud Application Manager. They are discovered in the synchronization event of a provider and are classified by Type and Subtype.
-  * Type: One of Compute, Network, Database, Storage or Other
-  * Subtype: this is the instance class, specific to each provider type. For example, for AWS type providers we can see VPC or Application Load Balancers (for Network type instances), and S3 or Elastic Block Storages (for Storage type instances)
+  * Type: Any of Compute, Network, Database, Storage or Other
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md]:
```suggestion
  * Type: One of Compute, Network, Database, Storage or Other
```

**Pull request merged**